static/js/apps/eval_retrieval_generation/sxs/query_with_tables.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useContext } from "react";
```

Comment:
add copyright comment & a file level comment describing the component

Reply:
Done.